packages/table/src/locator.ts

```diff
@@ -49,17 +49,52 @@ export interface ILocator {
 export class Locator implements ILocator {
     private static CELL_HORIZONTAL_PADDING = 10;
 
+    private grid: Grid;
+
+    private rowHeaderWidth = 0;
+    private columnHeaderHeight = 0;
+
+    private numFrozenRows = 0;
+    private numFrozenColumns = 0;
+
     public constructor(
         private tableElement: HTMLElement,
         private bodyElement: HTMLElement,
-        private grid: Grid,
     ) {
+        // empty constructor
     }
 
+    // Setters
+    // =======
+
     public setGrid(grid: Grid) {
         this.grid = grid;
+        return this;
```

Comment:
Both work, I like the second kind slightly better? But I don't have a strong preference in either direction.